### PR TITLE
add exception for each convertImage params

### PR DESCRIPTION
**Pull request by ChristianTC**
Could you review it please

### CODE REVIEW

File: CONVERT_SERVICE/convert_service/convert_image/views.py [hunk 1]
```diff
@@ -25,26 +22,33 @@
 
 
 class ImageConverter(View):
+
```
Comment (awielern) [inline CONVERT_SERVICE/convert_service/convert_image/views.py]: Please add the description of the class and the method

File: CONVERT_SERVICE/convert_service/convert_image/views.py [hunk 2]
```diff
@@ -55,21 +59,35 @@ def post(self, request):
...
+                result_error = {
+                    "status": "ERROR",
+                    "imageOutput": "NOT IMAGE"
+                }
+                return HttpResponse(json.dumps(result_error), 'application/json')
 
         return HttpResponse("Please, used method POST")
+
+    @staticmethod
+    def except_request(request, value):
```
Comment (awielern) [inline CONVERT_SERVICE/convert_service/convert_image/views.py]: Please add the description of the method

**Review by Franco98404 — APPROVED** [review]
It's ok. 👍

File: CONVERT_SERVICE/convert_service/convert_image/model/convert_image.py [hunk 3]
```diff
@@ -9,23 +9,44 @@
...
+from django.http import HttpResponse
 from wand.image import Image
-from wand.display import display
 from wand.color import Color
 import datetime
+import json
 
 
 class ConvertImage:
     """Class to convert image with many params"""
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_image/model/convert_image.py]: Please may you clarify the class description?... be more specific.

File: CONVERT_SERVICE/convert_service/convert_image/model/file.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information").  You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+from django.core.files.storage import FileSystemStorage
+from pathlib import Path
+import json
+
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_image/model/file.py]: please delete this blank line

File: CONVERT_SERVICE/convert_service/convert_image/test/test_views.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
+
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_image/test/test_views.py:12]: please delete this blank line

File: CONVERT_SERVICE/convert_service/convert_image/test/test_views.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_image/test/test_views.py]: please add copyright information

**Review by lcarrietas — COMMENTED** [review]
Please check comments

File: CONVERT_SERVICE/convert_service/convert_image/test/test_views.py (hunk 5 above)
Comment (ChristianTC) [reply CONVERT_SERVICE/convert_service/convert_image/test/test_views.py:12]: resolved

File: CONVERT_SERVICE/convert_service/convert_image/model/convert_image.py (hunk 3 above)
Comment (ChristianTC) [reply CONVERT_SERVICE/convert_service/convert_image/model/convert_image.py]: resolved

**Review by lcarrietas — APPROVED** [review]
Good

**Pull request merged**